src/Components/App/App.js

```diff
@@ -1,20 +1,52 @@
 import './App.scss';
-import {Homepage} from '../HomePage/Homepage'
-import Header from '../Header/Header'
-import React, {Component} from 'react'
-import {Route} from 'react-router-dom'
+import {Homepage} from '../HomePage/Homepage';
+import Header from '../Header/Header';
+import { Form } from '../Form/Form';
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
 
 class App extends Component {
   constructor() {
-    super()
+    super();
+    this.state = {
+      activities: [],
+      musicAnswers:[],
```

Comment:
It might be more semantic to have the activity name changed to its plural form.  For example, `movieAnswers` instead of `moviesAnswers.` Just a thought, definitely not a big deal right now.